packages/material-ui/src/ButtonBase/ButtonBase.js

```diff
@@ -121,6 +123,10 @@ class ButtonBase extends React.Component {
     clearTimeout(this.focusVisibleTimeout);
   }
 
+  getButtonNode() {
+    return ReactDOM.findDOMNode(this.buttonRef.current);
```

Comment:
Smart.

Reply:
I need to verify if that is actually tested in react or if there's confirmation for that behavior. It could very well be accidental at which point the whole `findDOMNode` migration becomes a lot trickier.

Reply:
Can we ask somebody from the React team?

Reply:
I'm writing a test and will submit this as a PR to react. We'll see what they'll say about it.